doc/man7/des_modes.pod

```diff
@@ -16,7 +16,7 @@ other things.
 
 Normally, this is found as the function I<algorithm>_ecb_encrypt().
 
-=over 2
+=over 4
```

Comment:
Because *this* is an odd looking bullet list:

```
*   item 1
*   item 2
*   item 3
```

This is what `=over 2` gives:

```
* item 1
* item 2
* item 3
```

Like I said, I think the proper change is to make the other bullet lists `=over 2` as well.